AppCenter/AppCenter/Internals/Util/MSUtility+StringFormatting.m

```diff
@@ -12,8 +12,20 @@
  */
 NSString *MSUtilityStringFormattingCategory;
 
+// We support the following formats:
+// target=<..>
+// appsecret=<..>
+// target=<..>;appsecret=<..>
+// ios=<..>;macos=<..>
+// targetIos=<..>;targetMacos=<..>
+// ios=<..>;macos=<..>;targetIos=<..>;targetMacos=<..>
+
 static NSString *kMSTransmissionTargetKey = @"target=";
 static NSString *kMSAppSecretKey = @"appsecret=";
+static NSString *kMSAppSecretIosKey = @"ios=";
+static NSString *kMSAppSecretMacosKey = @"macos=";
```

Comment:
I think we can have a `kMSAppSecretOSKey` that we'd instantiate differently depending on the OS to simplify the logic:
```
#if TARGET_OS_IOS
static NSString *kMSAppSecretOSKey = @"ios=";
#elif TARGET_OS_OSX || TARGET_OS_MACCATALYST
static NSString *kMSAppSecretOSKey = @"macos=";
#endif
```